constants.py

```diff
@@ -666,6 +666,9 @@
 kIntakeCameraHeightInMeters = 0.5
 kIntakeCameraCenterOffsetInMeters = 0.2
 kIsIntakeCameraCentered = False
+kIntakeGearRatio = 36 / 24
+kIndexerGearRatio = 5 / 1
+kStagingGearRatio = 5 / 1
 
 kIntakeSpeed = 500
```

Comment:
Speeds should be changed to reflect new changes

Reply:
They're changed in the set

Reply:
right, but the speeds on the intake should be multiplied by like 6 and indexer/intake by 1.25 or so to account for all the speed changes to remain constant speeds from what we set

Reply:
What do you mean by "account for all the speed changes to remain constant speeds"? These make our constants, which are measured in rpm, to be the output rpm. That's what we wanted originally, no?